Mathlib/GroupTheory/QuotientGroup/Defs.lean

```diff
@@ -102,6 +106,11 @@ theorem ker_mk' : MonoidHom.ker (QuotientGroup.mk' N : G →* G ⧸ N) = N :=
   Subgroup.ext eq_one_iff
 -- Porting note: I think this is misnamed without the prime
 
+@[to_additive]
+theorem le_comap_mk' (N : Subgroup G) [N.Normal] (H : Subgroup (G ⧸ N)) :
+    N ≤ Subgroup.comap (QuotientGroup.mk' N) H := by
+  simpa using (Subgroup.comap_mono bot_le : (mk' N).ker ≤ Subgroup.comap (mk' N) H)
```

Comment:
I like the following way of hinting to `comap_mono` slightly more, since it makes more explicit which argument couldn't be inferred. (And I would argue `f` should be explicit in `comap_mono`!)
```suggestion
  simpa using Subgroup.comap_mono (f := mk' N) bot_le
```

Reply:
Suggestion adopted, thanks!